api/utils/helpers.py

```diff
@@ -1,9 +1,91 @@
-from flask import json, Response, request
+import json
+import os
+import logging
+from types import GeneratorType
+
 from sqlalchemy_utils import sort_query
 
 
+def responsify(body, code, headers=None):
+    default_headers = {"Access-Control-Allow-Origin": "*", "Access-Control-Allow-Credentials": True}
+    default_headers.update(headers) if headers else None
+    results_obj = {}
+    if isinstance(body, dict):
+        results_obj = body
+    elif isinstance(body, list) or isinstance(body, str):
+        results_obj = {"data": body}
+    elif isinstance(body, GeneratorType):
```

Comment:
We should be returning a list because list comprehension is much faster anyway. I'm fine with leaving this condition here, but it really shouldn't be hit if we can help it.